Applications/Resnet/PyTorch/transLayer_v2.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+##
+# Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+#
+# @file transLayer_v2.py
+# @date 19 October 2021
+# @brief Rewrite parameters in the order consistent with nntrainer for the torch model
+# @author Jihoon lee <jhoon.it.lee@samsung.com>
+
+import torch
+from collections.abc import Iterable
+from zoneout import Zoneout
```

Comment:
We don't need zoneout in resnet so we could delete this

Reply:
> We don't need zoneout in resnet so we could delete this

I will reflect your comment. thanks :)